modules/jfx.incubator.richtext/src/main/java/com/sun/jfx/incubator/scene/control/richtext/rtf/RTFReader.java

```diff
@@ -782,11 +782,11 @@ public void close() {
             HashMap<Integer, Style> secStyles = new HashMap<>();
             for (StyleDefiningDestination style : definedStyles.values()) {
                 Style defined = style.realize();
-                String stype = (String)defined.getAttribute(STYLE_TYPE);
+                Object stype = defined.getAttribute(STYLE_TYPE);
                 Map<Integer, Style> toMap;
-                if (stype.equals(STYLE_SECTION)) {
+                if (STYLE_SECTION.equals(stype)) {
```

Comment:
Since this is a marker object, should it be `==` rather than `.equals`? They are equivalent, so it's up to you.